cssstyles/home.css

```diff
@@ -0,0 +1,683 @@
+@import url('https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&family=Fira+Sans:ital,wght@1,500&family=Heebo&family=League+Gothic&family=Montserrat:wght@800&family=Open+Sans:wght@800&family=Poppins:wght@300&display=swap');
+
```

Comment:
- [ ] Please for headings use `Cocogoose` font family as required. For other small texts you used correct font family 👌🏼

![image](https://user-images.githubusercontent.com/69011963/176916243-38d75c1c-25d4-4494-93ce-87918a715e21.png)

Reply:
Please be sure to make this change so that your project meets the requirements. I understand that you cannot find the correct font on Google Fonts but luckily there is an alternate option. You can discover how to add a specific font to your website in the following [link](https://www.w3schools.com/css/css3_fonts.asp). You could search for the font file on google, download it, and follow the tutorial 🤓